include/net/websocket_console.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2017 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __WEBSOCKET_CONSOLE_H__
+#define __WEBSOCKET_CONSOLE_H__
+
+#include <net/websocket.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Websocket console library
+ * @defgroup websocket Websocket Console Library
```

Comment:
include/net/websocket.h also defines a defgroup called ``websocket``.  Should this be named maybe ``websocket_console``